chat_gpt_interface.py

```diff
@@ -0,0 +1,17 @@
+import openai
+import asyncio
+
+
+async def send_query(questions: str) -> list:
+
+    file = open('api_key.txt', 'r')
+    my_api_key = file.read()
+    openai.api_key = my_api_key
+    messages = [{'role': 'user', 'content': "Explain this slide of PowerPoint: " + ' '.join(questions)}]
+
+    # Generate the completion
```

Comment:
Add single line comments only when the code below them is complicated, or requires additional context